.travis.yml

```diff
@@ -1,40 +1,45 @@
-# The only current way of using OS X virtual machines in Travis CI
-# is setting up a generic environment
-# This is not a problem because Python will be installed through conda
-
-language: generic
-os: osx
-
-branches:
-  only:
-    - master
-    - 0.3.x
-    - 0.4.x
-    - 0.5.x
-    - 0.6.x
-    - 0.7.x
-    - 0.8.x
-    - 0.9.x
-    - 0.10.x
-    - 0.11.x
-    - 0.12.x
-
+language: python
+dist: xenial
 env:
   global:
-    - MPLBACKEND="Agg"
-
+    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - SEGFAULT_SIGNALS=all
+matrix:
+  include:
+    - python: '3.6'
+      env:
+        - TOXENV=check
+    - python: '3.6'
+      env:
+        - TOXENV=docs
+    - python: '3.5'
+      env:
+        - TOXENV=py35,report,codecov
+    - python: '3.6'
+      env:
+        - TOXENV=py36,report,codecov
+    - python: '3.7'
+      env:
+        - TOXENV=py37,report,codecov
+    - python: 'pypy3.5-6.0'
+      env:
+        - TOXENV=pypy3,report,codecov
 before_install:
-  - HOMEBREW_NO_AUTO_UPDATE=1 brew install python3
-  - python3 -m venv venv
-  - source venv/bin/activate
-  - python3 -m pip install --upgrade pip
-
+  - python --version
+  - uname -a
+  - lsb_release -a
 install:
-  - pip install numpy  # Required
-  - pip install .  # Test installation correctness
-  - pip install .[dev]  # Test development dependencies
-
+  - pip install tox
+  - virtualenv --version
+  - easy_install --version
```

Comment:
`easy_install` must burn in hell :) please remove this line so I don't have nightmares at night!